tests/resolvers/Mutation/updatePost.spec.ts

```diff
@@ -199,4 +207,61 @@ describe("resolvers -> Mutation -> updatePost", () => {
       );
     }
   });
+
+  it("throws error if title is provided and post is not pinned", async () => {
+    const { requestContext } = await import("../../../src/libraries");
+    vi.spyOn(requestContext, "translate").mockImplementationOnce(
+      (message) => message
+    );
```

Comment:
1. Pinned posts can only be created by Super Admins or Admins for the organization where the post was created.
2. Please make sure this logic is enforced